src/components/PopupWithDelete.js

```diff
@@ -0,0 +1,27 @@
+import Popup from "./Popup";
+
+export default class PopupWithDelete extends Popup {
+  constructor(popupSelector) {
+    super(popupSelector);
+
+    this._deleteCardForm =
```

Comment:
Needs correction: `PopupWithConfirmation` is a general class. It can be used for any confirmation, not only card delete confirmation, so you should remove all the card mentions

Reply:
I'm not sure I understand. You want me to declare those in the parent class?

Reply:
No, I mean remove all the mentions of the `card`, for example instead of `_deleteCardForm` variable name use `_deleteForm` or `_form`